docs/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@ All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 
+## v4.x.x.x - 2023-08-17 - [PR#972](https://github.com/NOAA-OWP/inundation-mapping/pull/972)
+
+Adds functionality to `tools/inundate_mosaic_wrapper.py` and incorporates functionality into existing `inundation-mapping` scripts.
+
+### Changes
+
+- `tools/`
```

Comment:
There are changes to `overlapping_inundation.py`  & `mosaic_inundation.py` which are not listed here. Could you please list that file, and describe the high level changes?